### PR TITLE
add bank uk_UA

### PR DESCRIPTION
**Pull request by SanderFtn**
add provider.bank for uk_UA localization

### CODE REVIEW

**Comment by stefan6419846** [comment]
Could you please add corresponding tests as well?

**Comment by SanderFtn** [comment]
Yes.
Add some tests with new commit.

File: tests/providers/test_bank.py [hunk 1]
```diff
@@ -136,6 +137,21 @@ def test_iban(self, faker, num_samples):
             assert re.fullmatch(r"\d{2}\d{24}", iban[2:])
 
 
+class TestUkUA:
```
Comment (fcurella) [inline tests/providers/test_bank.py:140]: The capitalization is actually meaningful here, because of how we run tests.

```suggestion
class TestUkUa:
```

**Review by fcurella — CHANGES_REQUESTED** [review]

**Review by fcurella — APPROVED** [review]
Thank you!

**Comment by SanderFtn** [comment]
Thank you too!

**Pull request merged**